src/js/pinny.js

```diff
@@ -100,49 +104,56 @@
+                setTimeout(function () {
+
+                    this._disableExternalInputs();
+                    this._focus();
+                    // Only run lockup if another pinny isn't open and has
+                    // locked up the viewport already
+                    if (this._activePinnies()) {
+                        this.$pinny.data('lockup')._trigger('locked');
+                    } else {
+                        // The other pinny has probably set this listener already
+                        $window.on(events.orientationchange, this._blurActiveElement.bind(this));
+                        this.$pinny.lockup('lock');
+                    }
 
-                EventPolyfill.on(events.resize, function() {
-                    plugin._repaint();
-                });
+                    EventPolyfill.on(events.resize, this._repaint);
 
-                if (!this._activePinnies() && iOS7orBelow) {
-                    $window.on(events.orientationchange, this._blurActiveElement.bind(this));
-                }
+                    this.$pinny
+                        .addClass(classes.OPENED)
+                        .attr('aria-hidden', 'false');
 
-                this.$pinny
-                    .addClass(classes.OPENED)
-                    .attr('aria-hidden', 'false');
+                    this.$container.attr('aria-hidden', 'true');
 
-                this.$container.attr('aria-hidden', 'true');
+                    this._trigger('opened');
 
-                this._trigger('opened');
+                }.bind(this), 0);
             },
             closeComplete: function() {
-                this.$pinny
-                    .removeClass(classes.OPENED)
-                    .attr('aria-hidden', 'true');
-
-                this._enableExternalInputs();
-                this._resetFocus();
-
-                // only unlock if there isn't another pinny
-                // that requires the viewport to be locked
-                if (!this._activePinnies()) {
-                    this.$pinny.lockup('unlock');
-                    EventPolyfill.off(events.resize);
-                    $window.off(events.orientationchange);
-                }
+                setTimeout(function () {
+                    this.$pinny
+                        .removeClass(classes.OPENED)
+                        .attr('aria-hidden', 'true');
+
+                    this._enableExternalInputs();
+                    this._resetFocus();
+
+                    // only unlock if there isn't another pinny
+                    // that requires the viewport to be locked
+                    if (this._activePinnies()) {
+                        this.$pinny.data('lockup')._trigger('unlocked');
+                    } else {
+                        this.$pinny.lockup('unlock');
+                        EventPolyfill.off(events.resize);
+                        $window.off(events.orientationchange);
```

Comment:
Nope, we're not! @fractaltheory pointed out that the actual event is namespaced.